.gitignore

```diff
@@ -8,3 +8,4 @@ __pycache__/
 attic/multibody/solver_log/
 .clwb/
 .csv
+.ijwb
```

Comment:
This is another IDE folder because I have pycharm installed on my system. Would love to hvae thi included in the master gitignore